qbft/spectest/tests/controller_spectest.go

```diff
@@ -216,6 +216,10 @@ func (test *ControllerSpecTest) overrideStateComparison(t *testing.T) {
 func (test *ControllerSpecTest) GetPostState() (interface{}, error) {
 	contr := test.generateController()
 
+	if test.StartHeight != nil {
+		contr.Height = *test.StartHeight
+	}
+
```

Comment:
I added it in the `Run` function but forgot to add it in the `GetPostState`
None of these tests failed but another test that I did for another PR (in which I tried to use the StartHeight) would fail (that's actually why I remembered about it)